Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt

```diff
@@ -74,8 +76,11 @@ fun SignIn(onNavigationEvent: (SignInEvent) -> Unit) {
         },
         content = {
             SignInSignUpScreen(
-                onSignedInAsGuest = { onNavigationEvent(SignInEvent.SignInAsGuest) },
-                modifier = Modifier.fillMaxWidth()
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .wrapContentWidth(align = Alignment.CenterHorizontally)
+                    .widthIn(max = 840.dp),
```

Comment:
Nit: Would you mind adding a comment about what this does? It'll be easier for people to pick up what's going on and potentially copy-paste this